lib/internal/modules/esm/resolve.js

```diff
@@ -2,11 +2,14 @@
 
 const {
   ArrayIsArray,
+  ArrayPrototypeIncludes,
   JSONParse,
   JSONStringify,
+  ObjectFreeze,
   ObjectGetOwnPropertyNames,
   ObjectPrototypeHasOwnProperty,
   SafeMap,
+  Set,
```

Comment:
```suggestion
  SafeSet,
```

Reply:
I don't think we want to directly expose `SafeSet` to users.

Reply:
To the user, we only expose the array. The `Set` is only passed into internal functions.

Reply:
k